src/main/java/fr/polytech/sim/log/LogFactory.java

```diff
@@ -0,0 +1,7 @@
+package fr.polytech.sim.log;
+
+public class LogFactory {
+    public static TimestampedLoggerDecorator createLogger(String name) {
```

Comment:
Le type de retour devrai être Logger pour garder la généraliter de cette méthode.
```suggestion
    public static Logger createLogger(String name) {
```